src/Management/src/OpenTelemetryBase/Exporters/Wavefront/WavefrontTraceExporter.cs

```diff
@@ -19,18 +19,29 @@ namespace Steeltoe.Management.OpenTelemetry.Exporters
     /// </summary>
     public class WavefrontTraceExporter : BaseExporter<Activity>
     {
-        private readonly ILogger<WavefrontMetricsExporter> _logger;
+        private readonly ILogger<WavefrontTraceExporter> _logger;
         private WavefrontDirectIngestionClient _wavefrontSender;
         private WavefrontExporterOptions _options;
 
-        public WavefrontTraceExporter(IWavefrontExporterOptions options, ILogger<WavefrontMetricsExporter> logger)
+        public WavefrontTraceExporter(IWavefrontExporterOptions options, ILogger<WavefrontTraceExporter> logger)
         {
             _options = options as WavefrontExporterOptions ?? throw new ArgumentNullException(nameof(options));
-            var token = _options.ApiToken ?? throw new ArgumentNullException(nameof(_options.ApiToken));
+
+            var token = string.Empty;
+            var uri = _options.Uri;
+            if (_options.Uri.StartsWith("proxy://"))
+            {
+                uri = "http" + _options.Uri.Substring("proxy".Length); // Proxy reporting is now http on newer proxies.
```

Comment:
Would there be a need for https here?

Reply:
Scratch that, I was pretty sure I got this from a Java implementation and found it: https://github.com/micrometer-metrics/micrometer/blob/350a89c43ae04522d21bd9b1e802196bbe0b04c1/implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontMeterRegistry.java#L312